ui/src/app/cohort-search/redux/actions/service.ts

```diff
@@ -178,62 +178,101 @@ export class CohortSearchActions {
     }
   }
 
-  removeGroup(role: keyof SearchRequest, groupId: string): void {
+  removeGroup(role: keyof SearchRequest, groupId: string, status?: string): void {
     const group = getGroup(groupId)(this.state);
-    this.debugLog(`Removing ${groupId} of ${role}:`);
-    this.debugDir(group);
-
     this.cancelIfRequesting('groups', groupId);
-
-    this._removeGroup(role, groupId);
-    this.removeId(groupId);
-
+    if (status) {
+      this.hideGroup(groupId, status);
+    } else {
+      this._removeGroup(role, groupId);
+      this.removeId(groupId);
+    }
     group.get('items', List()).forEach(itemId => {
       this.cancelIfRequesting('items', itemId);
-      this._removeGroupItem(groupId, itemId);
-      this.removeId(itemId);
+      if (!status) {
+        this._removeGroupItem(groupId, itemId);
+        this.removeId(itemId);
+      }
     });
-
-    const hasItems = !group.get('items', List()).isEmpty();
-    if (hasItems) {
-      this.requestTotalCount();
+    if (this.hasActiveItems(group)) {
+      const activeGroupsWithItems = !groupList('includes')(this.state)
+        .merge(groupList('excludes')(this.state))
+        .filter(
+          grp => grp.get('status') === 'active'
+            && grp.get('id') !== groupId
+            && this.hasActiveItems(grp)
+        )
+        .isEmpty();
+      if (activeGroupsWithItems) {
+        this.requestTotalCount();
+      } else {
+        // todo clear total count
+        this.clearTotalCount();
+      }
     }
   }
 
-  removeGroupItem(role: keyof SearchRequest, groupId: string, itemId: string): void {
+  removeGroupItem(
+    role: keyof SearchRequest,
```

Comment:
Seems we have a bug here when removing the last search group item the status is set to 'deleted' causing a 500 on group count and total count never getting called.

Reply:
Fixed by pausing the delete timer when the wizard is open, but that may change depending on feedback

Reply:
Removed pausing of delete timer. Will now keep empty groups when all items are deleted per recommendation of CB team